lib/faker/default/coin.rb

```diff
@@ -3,10 +3,28 @@
 module Faker
   class Coin < Base
     class << self
+      ##
+      # Retrieves a random coin from any country.
+      #
+      # @return [String]
+      #
+      # @example
+      #   Faker::Coin.name #=> "Brazilian Real"
+      #
+      # @faker.version 2.6.0
```

Comment:
These were added in 1.9.2 with 21057576d6a7236f63b0539183adf7c8c17f8256

```suggestion
      # @faker.version 1.9.2
```